frontend/src/domain/legalPeriodicalEdition.ts

```diff
@@ -21,9 +21,17 @@ export default class LegalPeriodicalEdition {
 
   constructor(data: Partial<LegalPeriodicalEdition> = {}) {
     Object.assign(this, data)
+    if (this.id === undefined) {
+      this.id = crypto.randomUUID()
+    }
   }
 
-  get missingRequiredFields() {
```

Comment:
Method naming: when defining a getter in js / ts, there is no need to use the get prefix in the method name itself. I renamed it on this [commit](https://github.com/digitalservicebund/ris-backend-service/commit/8ba4d1e0f241e89dac90bc6e52b683499ca47b5b)